src/core/jsproxy.c

```diff
@@ -3982,14 +3982,14 @@ JsProxy_get_subtype(int flags)
 }
 
 #define SET_FLAG_IF(flag, cond)                                                \
-  try {                                                                        \
-    if (cond) {                                                                \
-      type_flags |= flag;                                                      \
-    }                                                                          \
-  } catch (e) {                                                                \
-    IF_DEBUG(console.warn(`Error raised in SET_FLAG_IF(#flag, #cond)`));       \
```

Comment:
Can you comment it's no longer a concern?

Reply:
I will write more tests that pass various horrible stuff through this code to make sure it never throws. I guess the comment should be "The condition must never raise an error!"
But in fact it's better for tests if this fails than if it gets swallowed as @ryanking13 points out.

Reply:
Ok `test_revoked_proxy` makes every single step here throw and catch an error.